DigitalLearningSolutions.Web.Tests/Helpers/LoginClaimsHelperTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Helpers
+{
+    using System.Security.Claims;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Controllers;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class LoginHelperTests
+    {
+        [Test]
+        public void Delegate_user_forename_and_surname_set_correctly()
+        {
+            var delegateUser = UserTestHelper.GetDefaultDelegateUser(firstName: "fname", lastName: "lname");
+            var delegateLoginDetails = new DelegateLoginDetails(delegateUser);
+            var claims = LoginClaimsHelper.GetClaimsForSignIn(null, delegateLoginDetails);
+            claims.Should().Contain((claim) => claim.Type == CustomClaimTypes.UserForename);
+            claims.Should().Contain((claim) => claim.Type == CustomClaimTypes.UserSurname);
+            var forenameClaim = claims.Find((claim) => claim.Type == CustomClaimTypes.UserForename);
+            var surnameClaim = claims.Find((claim) => claim.Type == CustomClaimTypes.UserSurname);
+            forenameClaim.Value.Should().Be("fname");
+            surnameClaim.Value.Should().Be("lname");
+        }
+        
+        [Test]
+        public void User_without_email_should_have_empty_string_email_claim()

```

Comment:
You can shorten test names by removing `should` e.g. User_without_email_has_empty_string_email_claim.
Test names are often long, so shortening them is always nice, but it's not that big a deal 🤷